.taplo.toml

```diff
@@ -3,13 +3,12 @@ include = ["**/*.toml", "**/Cargo.toml", "Cargo.toml"]
 align_comments      = true
 align_entries       = true
 allowed_blank_lines = 1
+indent_entries      = true
```

Comment:
Woah, not sure about this. While any style is technically fine if it's enforced by tooling, this diverges significantly from pretty much every other `Cargo.toml` I've seen... I'd avoid it on a "least surprises" basis.

Reply:
woops, thanks for catching that.

Reply:
I just uncommented out things I had tried before, trying to get the alignment right on the object style (which I ended up just backing out anyway)

Reply:
fixed, thanks.